routes/candidates.js

```diff
@@ -270,6 +283,24 @@ router.post('/create/:rcs_id/:office_id', function (req, res) {
             logger.write(connection, req.session.cas_user, "CANDIDATE_CREATE", "Added " + cms_data.username +
                 " as a candidate for office #" + office_id);
 
+            let mailoptions = {
+                text: "Hello " + cms_data.first_name + ",\n \n" +
+                "Your profile on the Elections website has been activated.\n" +
+                "You can sign into the Elections website with your RCS ID and edit your page here: https://elections.union.rpi.edu/candidate/" + cms_data.username + " \n" +
+                "If you are running for more than one office, you will get this email every time a new office is added to your profile. \n" +
```

Comment:
We could just change the message to say which office you've been added to which sidesteps this whole issue. The router already uses the office_id so it's just a matter of writing a SQL statement that identifies the office name. I'm just not sure how to do that in this context of this app.